gcp_variant_transforms/beam_io/vcf_parser.py

```diff
@@ -41,7 +46,7 @@
 DEFAULT_PHASESET_VALUE = '*'  # Default phaseset value if call is phased, but
                               # no 'PS' is present.
 MISSING_GENOTYPE_VALUE = -1  # Genotype to use when '.' is used in GT field.
-
+FIRST_HEADER_LINE = '##fileformat=VCFv4.2\n'
```

Comment:
To clarify:  do all `header_lines` passed via `_init_with_header` include  a `\n`?